civihr_employee_portal/views/views_export/views_documents.inc

```diff
@@ -351,6 +351,13 @@ $handler->display->display_options['arguments']['target_contact_id']['summary_op
 $handler = $view->new_display('block', 'Documents Dashboard Manager block', 'block_1');
 $handler->display->display_options['defaults']['title'] = FALSE;
 $handler->display->display_options['title'] = 'Documents Manager';
+$handler->display->display_options['defaults']['access'] = FALSE;
+$handler->display->display_options['access']['type'] = 'role';
+$handler->display->display_options['access']['role'] = array(
+  55120974 => '55120974',
+  37588037 => '37588037',
+  57573969 => '57573969',
```

Comment:
These are roleID's that won't change right?
Is it not possible to use their human readable name equivalents?